Sources/TensorFlow/Operators/Math.swift

```diff
@@ -2435,6 +2435,32 @@ internal extension Tensor where Scalar: TensorFlowFloatingPoint {
 
 // TODO: Consider making the return type be generic over `FloatingPoint` types
 // so that `self`'s scalar type can be any `Numeric` type.
+extension Tensor {
+    /// Returns `true` if given axis is in the range `[-rank, rank)`
+    /// for the provided tensors.
+    @usableFromInline
+    internal func isAxisInRange<T: BinaryInteger>(_ axis: T) -> Bool {
+        let axis = Int(axis)
+        return axis >= -rank && axis < rank
+    }
+
+    /// Returns `true` if given axes are all in the range `[-rank, rank)`
+    /// for the provided tensors.
+    @usableFromInline
+    internal func areAxesInRange<T: BinaryInteger>(_ axes: [T]) -> Bool {
+        return !axes.contains(where: { !isAxisInRange($0) })
+    }
+
+    /// Returns `true` if the scalars of the given 1-D tensor are all in the
```

Comment:
Formatting: this project uses [a 100-column limit](https://github.com/tensorflow/swift-apis/blob/master/.swift-format#L3). Please fill lines up to 100 columns:
```swift
    /// Returns `true` if the scalars of the given 1-D tensor are all in the range `[-rank, rank)`.
```

Same for other newly added doc comments please. You can use [`swift-format`](https://github.com/apple/swift-format) to auto-format newly added code:
```console
$ swift-format Sources/TensorFlow/Operators/Math.swift
# Project-specific configuration file: https://github.com/tensorflow/swift-apis/blob/master/.swift-format
```